docs/source/release_notes/release-notes.rst

```diff
@@ -1497,6 +1497,16 @@ v0.47.0
     *    - Type
          - Change
 
+    *    - |improved|
+         - ErrorProne is enabled and not ignored on all AtlasDB projects. This means that AtlasDB can be whitelisted in the internal logging aggregator tool.
+           On v0.42.0:
```

Comment:
What do the different versions here mean?  If we're using this mark a point at which products can whitelist for logging we should make this clear rather than linking to PRs people have to read to understand the status